pkg/cedar/intake/schemas/easi_business_case.json

```diff
@@ -29,75 +29,153 @@
+          "examples": [
+            "CMS wants to automate this process"
+          ]
         },
         "businessOwner": {
-          "type": "string"
+          "type": "string",
+          "description": "Business owner of this request",
+          "examples": [
+            "John Doe"
+          ]
         },
         "businessSolutions": {
           "items": {
             "$schema": "http://json-schema.org/draft-04/schema#",
             "$ref": "#/definitions/EASIBusinessSolution"
           },
-          "type": "array"
+          "type": "array",
+          "description": "Array Business Solutions (preferred and alternatives)"
         },
         "cmsBenefit": {
-          "type": "string"
+          "type": "string",
+          "description": "How CMS will benefit from this effort",
+          "examples": [
+            "Reduce FTE hours and generate better end products"
+          ]
         },
         "initialSubmittedAt": {
-          "type": "string"
+          "type": "string",
+          "description": "Timestamp of when request was initially submitted",
+          "examples": [
+            "2022-02-17 07:34:43.405934+00"
+          ]
         },
         "intakeId": {
-          "type": "string"
+          "type": "string",
+          "description": "Unique ID of the intake associated with this business case",
+          "examples": [
+            "36b85781-169a-4539-aa66-916663d8118c"
+          ]
         },
         "lastSubmittedAt": {
-          "type": "string"
+          "type": "string",
+          "description": "Timestamp of when request was last submitted",
+          "examples": [
+            "2022-02-11 16:34:43.405934+00"
+          ]
         },
         "lifecycleCostLines": {
           "items": {
             "$schema": "http://json-schema.org/draft-04/schema#",
             "$ref": "#/definitions/EASILifecycleCost"
           },
-          "type": "array"
+          "type": "array",
+          "description": "Array of LifecycleCostLines (costs associated with upcoming Fiscal Years)"
         },
         "priorityAlignment": {
-          "type": "string"
+          "type": "string",
+          "description": "The ways this effort align with organizational priorities",
+          "examples": [
+            "Aligns with CMS' automation push"
+          ]
         },
         "projectName": {
-          "type": "string"
+          "type": "string",
+          "description": "Name of the project",
+          "examples": [
+            "Easy Access to System"
+          ]
         },
         "requester": {
-          "type": "string"
+          "type": "string",
+          "description": "Name of the requester",
+          "examples": [
+            "John Doe"
+          ]
         },
         "requesterPhoneNumber": {
-          "type": "string"
+          "type": "string",
+          "description": "Phone number of requester",
+          "examples": [
+            "410-123-4567"
```

Comment:
The requester phone number only gets populated through the business case and isn't a required field until the final one so there may not be a ton of examples of it. Here is what it looks like for me locally:

![image](https://user-images.githubusercontent.com/68014929/155143977-fd43a985-ee16-49b5-be05-a0c461418569.png)

Reply:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/15203744/155167145-8b36fb73-51cc-426c-a736-6e188d06d0d0.png">
Looks like it can also be without dashes, if entered that way.

Reply:
We probably want to include both formats (with and without dashes) as examples, then. Looks like the formatting is just having multiple `example` bits in the Go field tag, i.e. `example=410-123-4567,example=1234567890`